os/arch/arm/src/amebasmart/amebasmart_spi.c

```diff
@@ -234,7 +234,7 @@ static struct amebasmart_spidev_s g_spi0dev = {
 	.spi_sclk = PB_6,
 	.spi_cs0 = PB_5,
 #if defined(CONFIG_SPI_CS) && defined(CONFIG_AMEBASMART_SPI0_CS)
-	.spi_cs1 = PB_26,
```

Comment:
Don't we have a provision to select the Pins from config?

Reply:
No, we don't have it as of now. Once the board configuration is fixed, we will not need to change the pins. So, I am not sure if it's really required.